internal/pkg/cli/app_package.go

```diff
@@ -157,12 +176,39 @@ func (opts *PackageAppOpts) getTemplate(env *archer.Environment) (string, error)
 			Env:      env,
 			ImageTag: opts.Tag,
 		})
-		return stack.Template()
+		tpl, err := stack.Template()
```

Comment:
What do you think of returning an interface of Template and Params, so you don't need this double string return (dyslexic me has a hard time with double string returns). Or a slice of typed structs {filename, content}

Reply:
👍  I like the latter, created a struct that groups the two templates